sktime/classification/compose/_ensemble.py

```diff
@@ -460,6 +462,10 @@ def _set_oob_score(self, X, y):
 
         self.oob_score_ = oob_score / self.n_outputs_
 
+    # TODO - Implement this abstract method properly.
+    def _set_oob_score_and_attributes(self, X, y):
+        pass
```

Comment:
raise NotImplementedError("Not implemented")

Reply:
Done 👍